crates/ruff/src/checkers/tokens.rs

```diff
@@ -117,7 +117,7 @@ pub fn check_tokens(
     // E701, E702, E703
     if enforce_compound_statements {
         diagnostics.extend(
-            pycodestyle::rules::compound_statements(tokens)
+            pycodestyle::rules::compound_statements(autofix, tokens)
```

Comment:
Nit: lets make `autofix` the second argument, to match the pattern below (`flake8_quotes::rules::from_tokens`, etc.).

I think we also need to pass in `settings`, because we have to verify that this _rule_ has autofix enabled. (`flake8_quotes::rules::from_tokens` probably has an example of that internally.)